src/main/java/net/atos/zac/flowable/FlowableService.java

```diff
@@ -226,6 +228,29 @@ public Task assignTask(final String taskId, final String userId) {
         return readTask(taskId);
     }
 
+    /**
+     * Assign task to a group
+     *
+     * @param taskId  Id of the task to assign
+     * @param groupId Id of the new group
+     * @return Assigned Task
+     */
+    public Task assignTaskToGroup(final String taskId, final String groupId) {
+        final Task task = readTask(taskId);
+        final String currentGroupId = task.getIdentityLinks().stream()
+                .filter(identityLinkInfo -> IdentityLinkType.CANDIDATE.equals(identityLinkInfo.getType()))
+                .findAny()
+                .map(IdentityLinkInfo::getGroupId)
+                .orElse(null);
+
+        if (currentGroupId != null) {
+            cmmnTaskService.deleteGroupIdentityLink(taskId, currentGroupId, IdentityLinkType.CANDIDATE);
```

Comment:
Je kun ook in 1 keer door streamen ...

task.getIdentityLinks().stream()
.filter(identityLinkInfo -> IdentityLinkType.CANDIDATE.equals(identityLinkInfo.getType()))
.map(IdentityLinkInfo::getGroupId)
.foreach(currentGroupId -> cmmnTaskService.deleteGroupIdentityLink(taskId, currentGroupId, IdentityLinkType.CANDIDATE))